packages/asset-swapper/src/types.ts

```diff
@@ -192,6 +193,10 @@ export interface SwapQuoteOrdersBreakdown {
  */
 export interface SwapQuoteRequestOpts extends CalculateSwapQuoteOpts {
     gasPrice?: BigNumber;
+    takerAddress?: string;
+    apiKey?: string;
+    enableRfqt?: boolean;
```

Comment:
This doesn't seem to be used anywhere.

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/commit/23684285877f9bf12294056b0fad1a1d9b034358